lib/fail.js

```diff
@@ -23,14 +23,19 @@ export default async function fail(pluginConfig, context, { Octokit }) {
     githubApiPathPrefix,
     githubApiUrl,
     proxy,
-    failComment,
     failTitle,
+    failComment,
+    failCommentCondition,
     labels,
     assignees,
   } = resolveConfig(pluginConfig, context);
 
   if (failComment === false || failTitle === false) {
     logger.log("Skip issue creation.");
+    logger.error(`Failure reporting should be disabled via 'failCommentCondition'.
+      Using 'false' for 'failComment' or 'failTitle' is deprecated and will be removed in a future major version.`);
+  } else if (failCommentCondition === false) {
+    logger.log("Skip issue creation.");
   } else {
```

Comment:
...when `failCommentCondition` is set to `"<% return false; %>"`, this block isn't executed, and I believe its because the template isn't resolved to `false` at that point yet..